server/middlewares/fetch-user.js

```diff
@@ -62,15 +61,7 @@ function getEventsForUserId(client, user_id) {
               properties,
               event_source: source,
               event_type: type,
-              context: {
-                location: {
-                  latitude: location.lat,
-                  longitude: location.lon
-                },
-                page: {
-                  url: context.page_url
-                }
-              }
+              context
```

Comment:
What's in context? Are we sure it's ok to return the whole thing?

Reply:
It's the return object from the User Events we index. It's an object that has a fixed shape: https://github.com/hull/hull/commit/664019fd275b532c370b99ff475197ae09f3c2b0